seed/migrations/0138_auto_20210512_0024.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 2.2.20 on 2021-05-12 00:24
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orgs', '0017_auto_20210218_1715'),
+        ('seed', '0137_statuslabel_show_in_list'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DerivedColumn',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('inventory_type', models.IntegerField(choices=[(0, 'Property'), (1, 'Tax Lot')])),
+                ('expression', models.CharField(max_length=526)),
+                ('organization', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='orgs.Organization')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='DerivedColumnParameter',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('parameter_name', models.CharField(max_length=50)),
+                ('derived_column', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='seed.DerivedColumn')),
+                ('source_column', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='seed.Column')),
```

Comment:
Not sure if this is the relevant line, but a question comes to mind here: What happens if a column is deleted that was associated to a DerivedColumn via a reference from this DerivedColumnParameter?

Reply:
Great point, I'm going to use `PROTECT` on this for now until we decide how we want to handle this.
With `CASCADE` the `evaluate` method is just going to end up returning `None` every  time b/c it's missing that parameter for all inventory.

Reply:
See
- https://github.com/SEED-platform/seed/pull/2682/files#diff-fc0d01e7c8f83cc5d37ed1e01da02c0b9c6998a458e79e9f08eccf4f32626899R31
- https://github.com/SEED-platform/seed/pull/2682/files#diff-bfbe128f201eda3032ef868db0ae6d82527e6352043c9bcbf68e97ed3b81bc16R315